src/open_inwoner/cms/cases/views/status.py

```diff
@@ -369,8 +369,11 @@ def handle_end_statustype_data(
         end_statustype_data = None
         if not status_types_mapping.get(end_statustype.url):
             end_statustype_data = {
-                "label": status_translate(
-                    end_statustype.omschrijving, default=_("No data available")
+                "label": (
+                    end_statustype.statustekst
+                    or status_translate(
+                        end_statustype.omschrijving, default=_("No data available")
+                    )
```

Comment:
This doesn't match the Taiga task:

>  statustekst is to be used if filled in, otherwise fallback to the omschrijving. The resulting text can then be mapped to a different one via the status mapping tabel

So I'd expect something like this here:

    status_translate((end_statustype.statustekst or end_statustype.omschrijving), default=_("No data available")))

This applies to all changes that involve the translation.